README.md

```diff
@@ -1,2 +1,20 @@
 # privacy-security
-Privacy &amp; Security Principles, Documents and Testing
+
+We believe that transparency in our software is an important part of ensuring security and privacy. This repository is meant to provide a public place for OSS developers, interested security and privacy experts to go to understand Path Check's:
```

Comment:
Pet peeve of mine: consider spelling out abbreviations or writing "Open Source Software (OSS)" the first time they're encountered in a document - while many seem obvious, there's a lot of them, and (presumably) a lot of new people reading any of our docs.